src/Model/CrashContext.cs

```diff
@@ -24,6 +32,8 @@ public CrashContext(DbContextOptions<CrashContext> options, ILogger<CrashHub> lo
 		/// <summary>The Latest Changes</summary>
 		public DbSet<MutableChange> LatestChanges { get; set; }
 
+		public DbSet<ManageableUser> ManageableUsers { get; set; }
+
 		/// <summary>The Users</summary>
 		public DbSet<User> Users { get; set; }
```

Comment:
Worth Explaining the differences between these 2.
One is Users editable by Admins, the other is users connecting.